Evaluation/Input/Content/Section2.3_Model_Parameters_and_Assumptions.md

```
@@ -0,0 +1,58 @@
+
+### Metabolism and Elimination <a id="model-parameters-and-assumptions-metabolism-and-elimination"></a>
+
+Dextromethorphan-O-demethylation via CYP2D6 leads to the formation of the major active metabolite dextrorphan. Dextrorphan subsequently undergoes rapid glucuronidation via uridine diphosphate-glucuronosyltransferases 2B (UGT2Bs), namely UGT2B15, or N-demethylation via CYP3A4. [Figure 1] (#Figure-1) shows the implemented metabolic pathways.
+
+**Figure 1: Implemented dextromethorphan metabolic pathways.**
+![Figure 1](images/Figure_1.PNG)
+<a id="figure-1"></a>
+
+Alternatively, dextromethorphan is N-demethylated by CYP3A4, which was found to be the main pathway of dextromethorphan metabolism in CYP2D6 PMs. Depending on the CYP2D6 phenotype, up to 50% of orally administered dextromethorphan is excreted unchanged in urine. For dextromethorphan and Dextrorphan, a GFR fraction of 1 was assumed. Dextrorphan O-glucuronide is renally eliminated via passive glomerular filtration and active secretion to the urine, a GFR fraction > 1 was optimized and represents both processes [Section 2.3.4](#parameter-identification).
+
+The principal pathway of dextromethorphan metabolism is the CYP2D6-mediated O-demethylation, leading to the formation of dextrorphan. This pathway was implemented using Michaelis-Menten kinetics. Cytochrome P450 2D6 (CYP2D6) was implemented in accordance with literature, using the PK-Sim® expression database to define their relative expression in the different organs of the body [PK-Sim Ontogeny Database Version 7.3](#references). The CYP2D6, CYP3A4 and UGT2B15 expression profiles are based on high-sensitive real-time RT-PCR [Nishimura 2003](#references). The t<sub>1/2<sub> for CYP3A4 was decreased from the default 37 h to 36 h and the reference concentration for UGT2B15 was increased from 1 µmol/L to 2.48 µmol/L for a better fit of the clinical data.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)